### DISCUSSION TITLE
Wolf CHA Wärmepumpe lesen

### DISCUSSION THREAD

**Discussion by noottzz** [zivillian/ism7mqtt#93] [General]
Hallo, wir haben eine Wolf Wärmepumoe CHA mit BM2 angeschlossen und können die Parameter auslesen. Mein Hauptinteresse ist, die Laufzeiten und energieverbräuche zu loggen, die auch im Smartset angezeigt werden, auf einer anderen Seite. Hat jemand dazu einen Zugang gefunden?

**Comment by drose28357** [comment]
Nee, loggen kann ich auch nicht. Mit dem Issue #126 (oder auch #127) sollten mehr Parameter für Long Term Statistic (LTS) aktiviert werden. Funktioniert bei mir aber nicht.
Ich kann z.B. sensor.wolf_cha_0x3_270049_leistungsaufnahme_wp_ehz nicht in einem Statistik-Diagramm verwenden.

In der Entwicklerwerkzeuge/zustände (develloper tools/states) ist für diese Entity keine LTS relevanten merkmale angelegt:
```
unit_of_measurement: kW
friendly_name: Wolf CHA Leistungsaufnahme (WP + EHZ)
```

Es fehlen:
`state_class: measurement
unit_of_measurement: kW`


Ich verwende die HA integration von [hassio-addon-ism7mqtt](https://github.com/b3nn0/hassio-addon-ism7mqtt)
